website/content/en/_index.html

```diff
@@ -22,20 +22,43 @@
 </p>
 {{% /blocks/lead %}}
 {{< blocks/section color="dark" type="row" >}}
-{{% blocks/feature icon="fa-solid fa-code" title="Install Lima" %}}
-  <a class="mr-3 mb-4 text-secondary" href="{{< relref "/docs/installation" >}}">
-Get Lima Here <i class="fas fa-arrow-alt-circle-right ml-2"></i></a>
-{{% /blocks/feature %}}
-
-{{% blocks/feature icon="fa-solid fa-chalkboard" title="Try Lima" %}}
-<a class="mr-3 mb-4 text-secondary" href="{{< relref "/docs/usage" >}}">
-Get started with Lima <i class="fas fa-arrow-alt-circle-right ml-2"></i></a>
-{{% /blocks/feature %}}
-
-{{% blocks/feature icon="fa-brands fa-github" title="Contributions welcome" %}}
-<a class="mr-3 mb-4 text-secondary" href="{{< relref "/community" >}}">
-Join the Lima community <i class="fas fa-arrow-alt-circle-right ml-2"></i></a>
-{{% /blocks/feature %}}
+
+  <div class="col-lg-4 mb-5 mb-lg-0 text-center">
```

Comment:
This has lot of duplicates now. 
Can you try adding url attribute to `{{% blocks/feature `. See if that becomes clickable 

Source https://www.docsy.dev/docs/adding-content/shortcodes/#blocksfeature

Reply:
I agree about the duplicates as well. I tried a number of options to do away with them, but with no success. I'm considering exploring the option of creating a custom block to minimise the duplication.

The option suggested with the link you've shared seems to just add a "Read more..." link to the **{{% blocks/feature** 
 without making the block/icons clickable.

<img width="757" alt="Screenshot 2023-10-03 at 08 53 54" src="https://github.com/lima-vm/lima/assets/26789177/13e8b1fb-e767-4b2e-85b9-d485bcc4e55d">

Reply:
Ah okay. In that case, do create a custom shortcode to minimise the duplication

We already have some custom defined shortcode here 
https://github.com/lima-vm/lima/blob/master/website/layouts/shortcodes/blocks

Reply:
Sure thing, I'd already taken a look through some of them. 
Let me try a custom shortcode and I'll revert afterwards.